### ISSUE TITLE
selection and user input background color is off after theme update

### ISSUE THREAD

**Issue by valentin-krasontovitsch** [github/copilot-cli#2180]
### Describe the bug

now i'm not a 100% sure where the problem is, as usual with colors, but i'm running on macos, in alacritty.

i have chosen a dark theme (cattpuccin-mocha) in alacritty, and have started a copilot cli session, and then i live switch themes to cattpuccin-latte. in general, the copilot session updates its colors seemingly OK, but the text that i write to copilot, after i submit it, gets a dark (black?) background.
while in a new copilot session (with still the light theme in alacritty) it gets a pleasant light grey background.
likewise, when i select text with my mouse, in the preexisting session the background is (almost) black, while in the new session it's a light purple.
i've attached a screenshot.

<img width="1872" height="831" alt="Image" src="https://github.com/user-attachments/assets/3a1359ca-25be-4bbc-a41b-1c65aa7ddba0" />

### Affected version

GitHub Copilot CLI 1.0.9

### Steps to reproduce the behavior

1. start alacritty with theme catpuccin-mocca
2. start copilot inside
3. switch themes in alacritty to catpuccin-latte
4. see that copilot theme get's messed up

setup of live reload of alacritty theme:
minimal alacritty config at `~/.config/alacritty/alacritty.toml`

```
[general]
live_config_reload = true
import = ["~/.config/alacritty/chosen_theme.toml"]
```

themes downloaded from https://github.com/catppuccin/alacritty.git into `~/.config/alacritty/catppuccin`

chosen_theme.toml is a relative symlink pointing to either `catppuccin/catppuccin-latte.toml` or `catppuccin/catppuccin-mocha.toml`

live theme switching happens by updating the symlink, and then touching the main alacritty config file `~/.config/alacritty/alacritty.toml`

### Expected behavior

colors should update in the copilot session 🌈

### Additional context

running in bash

**Comment by valentin-krasontovitsch** [comment]
work-around is simply issuing a /restart command
